crate_universe/src/rendering.rs

```diff
@@ -866,6 +906,7 @@ mod test {
             mock_render_config(None),
             mock_supported_platform_triples(),
             true,
+            true,
```

Comment:
I must be reading something wrong, but I don't see a `new` which takes two booleans? Does this need updating? (And throughout)